### PR TITLE
[docs] Fix typo

### PR DESCRIPTION
**Pull request by bmorelli25**
## What does this pull request do?

This fixes a couple typos in `api.asciidoc`.

## Checklist
~~- [ ] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).~~
~~- [ ] My code follows the style guidelines of this project (See `.rubocop.yml`)~~
~~- [ ] I have rebased my changes on top of the latest master branch~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] New and existing [**unit** tests](https://github.com/elastic/apm-agent-ruby/blob/master/CONTRIBUTING.md#testing) pass locally with my changes~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have updated [CHANGELOG.asciidoc](CHANGELOG.asciidoc)~~
~~- [ ] I have updated [supported-technologies.asciidoc](docs/supported-technologies.asciidoc)~~
~~- [ ] Added an API method or config option? Document in which version this will be introduced~~

### CODE REVIEW

**Comment by mikker** [comment]
These are actually meant to be spelled like `#method_name` as it's common in Ruby land to do so for instance methods. Adding the code ` ticks is a good idea, though.

File: docs/api.asciidoc [hunk 1]
```diff
@@ -425,13 +425,13 @@ end
 
 [float]
 [[api-transaction-sampled_]]
-==== #sampled?
+==== Sampled?
```
Comment (mikker) [inline docs/api.asciidoc]:
```suggestion
==== `#sampled?`
```

File: docs/api.asciidoc [hunk 2]
```diff
@@ -425,13 +425,13 @@ end
...
 
 Whether the transaction is _sampled_ eg. it includes stacktraces for its spans.
 
 [float]
 [[api-transaction-ensure_parent_id]]
-==== #ensure_parent_id
+==== `ensure_parent_id`
```
Comment (mikker) [inline docs/api.asciidoc]:
```suggestion
==== `#ensure_parent_id`
```

**Comment by bmorelli25** [comment]
> These are actually meant to be spelled like #method_name as it's common in Ruby land to do so for instance methods. Adding the code ` ticks is a good idea, though.

Ahh, thanks. I saw the `#` and immediately thought markdown instead of asciidoc. Didn't even consider that it was supposed to be named like that. Sorry 😨

**Comment by mikker** [comment]
Don’t worry 😊 appreciate the thought!

**Pull request merged**